### PR TITLE
Make compatible with AMI

### PR DESCRIPTION
**Pull request by ErinBecker**
This PR makes this lesson compatible with being taught using the Data Carpentry genomics lesson AMI. 

- [x] updates vcf cleaning script to use directory structure on AMI
- [x] removes data files from lesson repo
- [x] updates lesson files to use AMI directory structure (and read in data file silently from FigShare)

Changes elsewhere that support this PR:
- [x] VCF cleaning script and tidy VCF file have been added to [the Figshare publication](https://figshare.com/articles/Data_Carpentry_Genomics_beta_2_0/7726454) for this curriculum 
- [ ] creating new AMI that includes VCF cleaning script and tidy VCF file (directly from Figshare with `wget`)
- [ ] update RStudio on AMI and enable

### CODE REVIEW

**Comment by ErinBecker** [comment]
Closes https://github.com/carpentrieslab/genomics-r-intro/issues/14

**Comment by ErinBecker** [comment]
I don't have the necessary privileges to install or update software on the AMI. @fmichonneau - could you point me in the right direction?

> Last login: Mon Mar 18 20:02:52 2019 from 104.220.40.60
(base) dcuser@ip-172-31-57-71:~$ sudo apt-get install r-base
[sudo] password for dcuser: 
dcuser is not in the sudoers file.  This incident will be reported.
(base) dcuser@ip-172-31-57-71:~$ apt-get install r-base
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?

**Comment by naupaka** [comment]
Hi @ErinBecker -- should I hold off on a review until the steps listed above are completed?

**Comment by ErinBecker** [comment]
@naupaka - Nope. A review any time would be appreciated. Sorry for the unclarity.

File: _episodes_rmd/04-dplyr.Rmd [hunk 1]
```diff
@@ -25,8 +25,10 @@ source("../bin/chunk-options.R")
 knitr_fig_path("04-")
 ```
 
-```{r, echo = FALSE, purl = FALSE, message = FALSE}
-variants <- read.csv("../data/combined_tidy_vcf.csv")
+```{r, echo = FALSE, eval = TRUE, purl = FALSE}
+## silently read in CSV file from FigShare
+
+variants <- read.csv("https://ndownloader.figshare.com/files/14632895")
```
Comment (naupaka) [inline _episodes_rmd/04-dplyr.Rmd]: What's the rationale for loading the data directly from figshare vs having it on the AMI? More portability of the code, I guess?

Comment (ErinBecker) [reply _episodes_rmd/04-dplyr.Rmd]: My immediate rationale is that I needed it in order to make the lesson render, but yes, also portability :-)

**Comment by ErinBecker** [comment]
Thanks for the review!

**Pull request merged**